sig-TEMPLATE/sig-charter-template-prefilled.md

```diff
@@ -0,0 +1,32 @@
+# SIG YOURSIG Charter
+
+## Scope
+
+Include a 2-3 sentence summary of what work SIG TODO does. Imagine trying to
+explain your work to a colleague who is familiar with KubeVirt but not
+necessarily all of the internals.
+
+### In scope
+
+#### Code, Binaries and Services
+
+- Unit and e2e tests
+- Foo API and behaviors
+- Foo or Foo level Controller
+- Foo level performance and scalability
+- CI for bar
+- …
+
+### Out of scope
+
+Bar API and controller, because they are cluster and not node level.
+
+## Roles and Organization Management
+
+This sig follows adheres to the Roles and Organization Management outlined in [sig-governance]
```

Comment:
Are you going to also document a `sig-governance`?

Reply:
Gargh. Lemme remove this trace.
Not now, but eventually we want to have it I guess.